donut/static/js/groups/positions.js

```diff
@@ -179,16 +204,35 @@ $(document).ready(function() {
             }
         });
     });
-}); 
+    $('#submitPosHoldBtn').click(function(e) {
+        e.preventDefault();
+        $.ajax({
+            type: 'POST',
+            url: '/1/positions/' + $("#posIdHold").val() + '/',
```

Comment:
Don't think trailing slash is needed

Reply:
Its not _needed_ but flask does complain if its not there:
`
"'A request was sent to this URL (http://127.0.0.1:9001/1/positions/2) but a redirect was issued automatically by the routing system to "http://127.0.0.1:9001/1/positions/2/".  The URL was defined with a trailing slash so Flask will automatically redirect to the URL with the trailing slash if it was accessed without one.  Make sure to directly send your POST-request to this URL since we can\'t make browsers or HTTP clients redirect with form data reliably or without user interaction.\n\nNote: this exception is only raised in debug mode'"
`